src/nxt_controller.c

```diff
@@ -1931,10 +1931,27 @@ nxt_controller_cert_in_use(nxt_str_t *name)
                 continue;
             }
 
-            nxt_conf_get_string(value, &str);
+            if (nxt_conf_type(value) == NXT_CONF_ARRAY) {
+                n = nxt_conf_array_elements_count(value);
```

Comment:
You know I always like to reduce the scope of variables...

We could declare `n` here...